keccak256/src/permutation/rho_checks.rs

```diff
@@ -454,13 +447,9 @@ pub struct OverflowCheckConfig<F> {
 impl<F: Field> OverflowCheckConfig<F> {
     pub fn configure(
         meta: &mut ConstraintSystem<F>,
-        cols_to_copy: Vec<Column<Advice>>,
         step2_range_table: &RangeCheckConfig<F, STEP2_RANGE>,
         step3_range_table: &RangeCheckConfig<F, STEP3_RANGE>,
     ) -> Self {
-        for &col in cols_to_copy.iter() {
-            meta.enable_equality(col);
-        }
```

Comment:
I think we can keep these lines. They don't create new columns.

Reply:
But we don't need to enable equality for `cols_to_copy` as they're removed. So mantaining these lines will cause an error.